src/components/icon/icon.scss

```diff
@@ -1,24 +1,3772 @@
 
+@font-face {
+  font-family:"material";
+  src:url("/dist/material-font/material.eot");
+  font-weight:normal;
+  font-style:normal;
+}
+
+@font-face {
+  font-family:"material";
+  src:url("/dist/material-font/material.eot");
+  src:url("/dist/material-font/material.eot?#iefix") format("embedded-opentype"),
+    url("/dist/material-font/material.woff") format("woff"),
+    url("/dist/material-font/material.ttf") format("truetype"),
+    url("/dist/material-font/material.svg?#material") format("svg");
```

Comment:
We will want to make the `font-face` root directory configurable and then inject it into a style tag during run time so that we don't force our users to have a certain folder structure. Also perhaps we should call the font-family `"material-icons"` instead